src/stories/tag/Tag.tsx

```diff
@@ -39,7 +49,23 @@ export const Tag: FC<TagProps> = ({
             size === 'compact' && typography.labelXXSmall,
             className,
         )}
+        data-status={colorLevel}
     >
-        {children}
+        {size !== 'small' && (
+            <div>
+                {children}
```

Comment:
children should still be required/visible for icon tags for accessibility purposes. you can throw it in a `VisuallyHidden`

Reply:
can refer to `<Button shape="circle">` for example implementation

Reply:
Okay used `VisuallyHidden`, though for reference the button uses a different method instead, but `VisuallyHidden` is used in places like the drawer and dialog titles